### ISSUE TITLE
Boolean defaults in parsed schema are parsed as `:True`/`:False` rather than `true`/`false`

### ISSUE THREAD

**Issue by markgdawson** [walmartlabs/lacinia#378]
When a schema is parsed that contains a Boolean with a default, this is parsed into a Lacinia EDN schema as `:True` or `:False` rather than `true` or `false`.

This can be verified with the followig

```clojure
(schema-parser/parse-schema "schema {
  query: Query
}

type Query {
  example(in: Boolean = False) : Boolean
  anotherExample(in: Boolean = True) : Boolean
}")
```

which produces:
```clojure
{:roots {:query :Query},
 :objects
 {:Query
  {:fields
   {:example
    {:type Boolean,
     :args {:in {:type Boolean, :default-value :False}}},
    :anotherExample
    {:type Boolean,
     :args {:in {:type Boolean, :default-value :True}}}}}}}
```
This results in associated resolvers getting the keywords `:True` or `:False` if nothing was provided and `true`/`false` otherwise.

This [issue seems to suggest](https://github.com/walmartlabs/lacinia/issues/143) that it would be appropriate to parse this to `true`/`false`.

**Comment by hlship** [comment]
This is not a bug.  Boolean values are `true` and `false`; the capitalization is what triggers the incorrect behavior.

```
(schema-parser/parse-schema "schema {
  query: Query
}

type Query {
  example(in: Boolean = false) : Boolean
  anotherExample(in: Boolean = true) : Boolean
}")
=>
{:roots {:query :Query},
 :objects {:Query {:fields {:example {:type Boolean, :args {:in {:type Boolean, :default-value false}}},
                            :anotherExample {:type Boolean, :args {:in {:type Boolean, :default-value true}}}}}}}

```

Also see [the spec](https://spec.graphql.org/June2018/#sec-Boolean-Value).

**Comment by markgdawson** [comment]
The schema would fail parsing unless true/false was capitalised in 0.37.0. But it seems that this has been fixed in 0.38.0.

Apologies for opening a stale issue.
